craft_application/application.py

```diff
@@ -446,11 +448,18 @@ def _try_load_project(
             self.services.project = self.get_project(
                 platform=platform, build_for=build_for
             )
-        except Exception:  # noqa: BLE001
+        except FileNotFoundError:
             if command.always_load_project == "try":
                 craft_cli.emit.debug("Project not found. Continuing without.")
             else:
                 raise
+        except (pydantic.ValidationError, ValueError, yaml.error.YAMLError) as exc:
+            if command.always_load_project == "try":
+                craft_cli.emit.progress(
+                    "Error loading project. Attempting to continue without a project.",
+                    permanent=True,
+                )
```

Comment:
why a perm message?

Reply:
For the remote-build use case, I would say that we should still require being in a valid project